### ISSUE TITLE
Add Allen Institute references to README + doc-string

### ISSUE THREAD

**Issue by rmarkello** [rmarkello/abagen#49]
## The issue

The Allen Institute has some [citation policies](https://alleninstitute.org/legal/citation-policy/) that we should probably do a better job highlighting throughout our documentation and codebase! 

## Proposed solution

As it applies to the current toolbox, we should add the following publication to all Acknowledgments and References sections:

```
Hawrylycz, M.J. et al. (2012) An anatomically comprehensive atlas of the adult
human transcriptome, Nature 489: 391-399. doi:10.1038/nature11405
```

Furthermore, we should probably add a disclaimer in the License Information section of the README that all the data obtained from calling `abagen.fetch_microarray()` is copyrighted by Allen Institute. Adding the following would be a good start:

```
All microarray expression data is copyrighted under [non-commercial reuse
policies] (http://alleninstitute.org/legal/terms-use/) by the Allen Institute
for Brain Science (© 2010 Allen Institute for Brain Science. Allen Human Brain
Atlas. Available from: human.brain-map.org).
```

**Comment by JamesFrierson1** [comment]
Hi @rmarkello I'd be happy to contribute! I can add an additional direction in the Acknowledgments section of the README that states contributors should include the proper citation provided per The Allen Institute's policy.

And I can add the disclaimer you wrote into the License Information section as well.

Is this what you're looking for? Thanks!

**Comment by rmarkello** [comment]
Hi @JamesFrierson1 ! That would be awesome ! :raised_hands: 

I realize now that the text I wrote to add to the License Information is formatted in Markdown but would need to be updated to reStructuredText for the README. All that should mean is changing the format of the [hyperlink](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#hyperlink-references), but let me know if you have any questions about that!

**Comment by rmarkello** [comment]
Closed with #57!
